docs/development/airflow.md

```diff
@@ -10,6 +10,9 @@ This section describes how to set up a local Airflow server which will orchestra
 !!!warning macOS Docker memory allocation
     On macOS, the default amount of memory available for Docker might not be enough to get Airflow up and running. Allocate at least 4GB of memory for the Docker Engine (ideally 8GB). [More info](https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html#)
 
+!!!warning Making sure the Docker set up is up to date
```

Comment:
this was in another PR

Reply:
Yep — now resolved. I added a comment in the closed PR about this: https://github.com/opentargets/genetics_etl_python/pull/223#issuecomment-1794523985